configuration/ConfigHandler.cpp

```diff
@@ -0,0 +1,50 @@
+#include "ConfigHandler.hpp"
+#include <iostream>
+#include <fstream>
+#include "../yaml-cpp/include/yaml-cpp/yaml.h"
```

Comment:
CMake allows to setup proper include paths in the build system, this avoids doing relative import paths